src/citrine/informatics/design_candidate.py

```diff
@@ -202,3 +203,20 @@ class HierarchicalDesignCandidate(Serializable["HierarchicalDesignCandidate"]):
 
     def __init__(self):
         pass  # pragma: no cover
+
+
+class SampleSearchSpaceResultCandidate(Serializable["SampleSearchSpaceResultCandidate"]):
+    """A hierarchical candidate material generated by the Citrine Platform.
+
+    This class represents the candidate computed by a design execution.
+    """
+
+    uid = properties.UUID('id')
+    """:UUID: unique external Citrine ID of the material"""
+    execution_uid = properties.UUID('id')
+    """:UUID: unique external Citrine ID of the execution"""
+    material = properties.Object(HierarchicalDesignMaterial, "material")
+    """:HierarchicalDesignMaterial: the material returned by the design workflow"""
+
+    def __init__(self):
+        pass  # pragma: no cover
```

Comment:
Due to a tweak I made a little while ago, I'd expect this `__init__` is not necessary. If you're getting errors after deleting it, I can take a look.